connector_search_engine/models/se_binding.py

```diff
@@ -36,25 +27,44 @@ class SeBinding(models.AbstractModel):
         # TODO: shall we use 'restrict' here to preserve existing data?
         ondelete="cascade",
     )
-    sync_state = fields.Selection(
+    state = fields.Selection(
         [
-            ("new", "New"),
-            ("to_update", "To update"),
-            ("scheduled", "Scheduled"),
+            ("to_recompute", "To recompute"),
+            ("recomputing", "Recomputing"),
+            ("to_export", "To export"),
+            ("exporting", "Exporting"),
             ("done", "Done"),
-            ("to_be_checked", "To be checked"),
+            ("invalid_data", "Invalid Data"),
+            ("to_delete", "To Delete"),
+            ("deleting", "Deleting"),
         ],
-        default="new",
-        readonly=True,
+        default="to_recompute",
+        compute="_compute_state",
+        store=True,
+        readonly=False,
     )
-    date_modified = fields.Datetime(readonly=True)
+    date_recomputed = fields.Datetime(readonly=True)
     date_syncronized = fields.Datetime(readonly=True)
     data = fields.Serialized()
     data_display = fields.Text(
         compute="_compute_data_display",
         help="Include this in debug mode to be able to inspect index data.",
     )
-    active = fields.Boolean(default=True)
```

Comment:
wouldn't it be better to deactivate the binding once it is deleted in the search engine instead of deleting it ?

Reply:
No we can drop it, there is no data to conserve on se.binding, it will avoid to have a growing database

Reply:
agreed, if we want to be able to know if a record was binded this would better be done by having a chatter message on the many2many